### PR TITLE
Initial Vert.x server implementation

### PR DESCRIPTION
**Pull request by Sammers21**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,38 +83,43 @@ SOFTWARE.
     </site>
   </distributionManagement>
   <dependencies>
+    <dependency>
+      <groupId>com.artipie</groupId>
```
Comment (g4s8) [inline pom.xml:83]: @Sammers21 it can't be merged, we need to wait for module release first

File: pom.xml [hunk 2]
```diff
@@ -83,38 +83,43 @@ SOFTWARE.
...
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-library</artifactId>
-      <version>2.1</version>
-      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-library</artifactId>
```
Comment (g4s8) [inline pom.xml]: @Sammers21 it should be `org.hamcrest:hamcrest` (without library), see http://hamcrest.org/JavaHamcrest/distributables

File: pom.xml [hunk 3]
```diff
@@ -165,6 +170,14 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
```
Comment (g4s8) [inline pom.xml]: @Sammers21 you don't need to configure compiler plugin by yourself, it's done in parent pom

File: pom.xml [hunk 4]
```diff
@@ -181,7 +194,6 @@ SOFTWARE.
                 <exclude>findbugs:.*</exclude>
                 <exclude>duplicatefinder:.*</exclude>
                 <exclude>xml:/src/it/settings.xml</exclude>
-                <exclude>checkstyle:/src/test/resources-binary/.*</exclude>
```
Comment (g4s8) [inline pom.xml:184]: @Sammers21 qulice already configured in parent pom, you can remove this profile

File: src/main/java/com/artipie/vertx/VertxSliceServer.java [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+    /**
+     * The slice to be served.
+     */
+    private final Slice served;
+
+    /**
+     * The port to start server on.
+     */
+    private final Integer port;
```
Comment (g4s8) [inline src/main/java/com/artipie/vertx/VertxSliceServer.java:60]: @Sammers21 why `Integer` instead of `int`?

**Comment by yegor256** [comment]
@Sammers21 no unit tests?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@Sammers21 please check few comments

File: src/main/java/com/artipie/vertx/VertxSliceServer.java (hunk 5 above)
Comment (Sammers21) [reply src/main/java/com/artipie/vertx/VertxSliceServer.java:60]: It does not matter

**Comment by Sammers21** [comment]
@rultor, release tag=`0.1.0`

**Pull request merged**